src/gpio.rs

```diff
@@ -1076,6 +1082,40 @@ mod chip {
     pin!(Gpio38:38, Input, RTC:2, ADC1:2, NODAC:0, NOTOUCH:0);
     pin!(Gpio39:39, Input, RTC:3, ADC1:3, NODAC:0, NOTOUCH:0);
 
+    impl<MODE> RmiiClkPin for Gpio0<MODE>
```

Comment:
All of these protected with `#[cfg(esp32)]` as per above. You can also probably move them to the esp32-specific `mod chip` and then re-export them from there with `pub use...`

Reply:
they where already in the esp32 `mod chip`. I added the guard for risc..